test/functional/JIT_Test/src/jit/test/jitserver/JITServerTest.java

```diff
@@ -45,7 +48,7 @@ public class JITServerTest {
 	private static final int PROCESS_START_WAIT_TIME_MS = 2 * 1000;
 	private static final int PROCESS_TERMINATE_WAIT_TIME_MS = 2 * 1000;
 	private static final int SERVER_START_WAIT_TIME_MS = 5 * 1000;
-	private static final int CLIENT_TEST_TIME_MS = 45 * 1000;
+	private static final int CLIENT_TEST_TIME_MS = 5 * 1000;
```

Comment:
Doesn't this change affect the existing tests?

Reply:
I have now changed this to have 2 wait times.